STW.SubmissionApi.sln

```diff
@@ -0,0 +1,24 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Still not in the right place. Needs to be in `src`. See Sean's page on wiki https://dev.azure.com/defragovuk/DEFRA-STW-Integration/_wiki/wikis/STW-Integration.wiki/25425/Repository-Directory-and-File-Structure